examples/parse_file.rs

```diff
@@ -0,0 +1,45 @@
+extern crate bincode;
+extern crate libc;
+extern crate redland_rs;
+#[macro_use]
+extern crate unwrap;
+
+use libc::{c_char, fclose, fdopen, fgets};
+use redland_rs::{KvStorage, Model, Parser, Serializer, Uri};
+use std::ffi::CStr;
+use std::fs::File;
+use std::io::prelude::*;
+use std::os::unix::io::IntoRawFd;
+
+fn main() {
+
+    let file_path = "/home/yoga/rdf/sparql11/sparql11.ttl";
+    let rust_file = File::open(file_path).unwrap();
```

Comment:
Better to use `unwrap!(...)` in the test/examples code instead of just `.unwrap`. The macro provides us with more context for the error (most importantly, the file name and a line number where it happened).